src/mrpro/operators/LinearOperator.py

```diff
@@ -47,6 +48,71 @@ def H(self) -> LinearOperator:  # noqa: N802
         """Adjoint operator."""
         return AdjointLinearOperator(self)
 
+    def operator_norm(
+        self,
+        initial_value: torch.Tensor,
+        dim: Sequence[int] | None,
+        n_max_iterations: int = 64,
```

Comment:
i prefered the previous max_iterations.

Reply:
fine for me. I guess, however, that the convergence depends quite a lot on the considered operator. I tried with the nabla operator for `dim=(-4,-3,-2,-1)` and `dim=(-2,-1)` and the accuracy seems to be fine after 20 iterations.

Reply:
Re: depending what one considers to be "fine" :D

I now included a test using the finite differences operator as well, and for dim corresponding to a 1D, 2D, 3D and 4D kernels, 50 iterations are not enough to obtain to pass the test for absolute and relative tolerance 1e-2.

Reply:
Ok, so 64 it is..